utilities/installer/SAMPLE_host.cfg

```diff
@@ -15,11 +15,23 @@
 # Replace the < > with your information
 
 HOSTS:
-    <agent1>:
+    <IOS XR node fully qualified domain name>:
         roles:
             - agent
-        platform: cisco-7-x86_64
-        ip: <fully qualified domain name>
+        platform: cisco_ios_xr-6-x86_64
+        ip: <ip address>
```

Comment:
So basically the old behavior was overloaded in that the host "name" was not used for anything but 'ip' was used for both reachability and for bootstrapping puppet.conf. This old behavior did *not* work for my XRv9k instance, as it has an IP address but no DNS entry - so if I specified `ip` as `192.168.122.222` then Beaker could reach the node but then it got `certname:192.168.122.222` in puppet.conf, which didn't work. Conversely, if I specified `ip:my-node.domain.com` then Beaker couldn't reach the node since it's not in DNS. Hence why I changed it so the `ip` is still used for reachability (since this is built-in to Beaker) and the `name` is used for configuration.

All of which is a long-winded way to say - this *can* be a DNS name, but going by [the Beaker docs](https://github.com/puppetlabs/beaker/blob/master/docs/Creating-A-Test-Environment.md#optional-host-settings) it's intended to be an IP address.